osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -0,0 +1,525 @@
+                    AutoSizeAxes = Axes.Y,
+                    Width = 0.4f,
+                    Direction = FillDirection.Vertical,
+                    LayoutDuration = 1,
+                    LayoutEasing = EasingTypes.OutQuint,
+                    Padding = new MarginPadding(10) { Top = 25 },
+                    Children = new []
+                    {
+                        description = new MetadataSegment
+                        {
+                            HeaderText = "Description",
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                        },
+                        source = new MetadataSegment
+                        {
+                            HeaderText = "Source",
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                        },
+                        tags = new MetadataSegment
+                        {
+                            HeaderText = "Tags",
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                        }
+                    },
+                },
+                new FillFlowContainer
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Width = 0.6f,
+                    Direction = FillDirection.Vertical,
+                    Spacing = new Vector2(0,15),
+                    Padding = new MarginPadding(10) { Top = 0 },
+                    Children = new Drawable[]
+                    {
+                        new Container
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Colour = Color4.Black,
+                                    Alpha = 0.5f,
+                                },
+                                new FillFlowContainer
+                                {
+                                    RelativeSizeAxes = Axes.X,
+                                    AutoSizeAxes = Axes.Y,
+                                    Direction = FillDirection.Vertical,
+                                    Spacing = new Vector2(0,10),
+                                    Padding = new MarginPadding(15) { Top = 25 },
+                                    Children = new []
+                                    {
+                                        circleSize = new DifficultyRow
+                                        {
+                                            DifficultyName = "Circle Size",
+                                            AutoSizeAxes = Axes.Y,

```

Comment:
same `AutoSizeAxes` and `RelativeSizeAxes` in every case. just move to inside `DifficultyRow`?